src/main/java/frc/robot/subsystems/arm/ArmCommandBuilder.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import frc.robot.subsystems.GBCommandsBuilder;
+import frc.utils.utilcommands.InitExecuteCommand;
+
+import java.util.function.DoubleSupplier;
+import java.util.function.Supplier;
+
+public class ArmCommandBuilder extends GBCommandsBuilder {
+
+	private final Arm arm;
+
+	public ArmCommandBuilder(Arm arm) {
```

Comment:
```suggestion
	ArmCommandBuilder(Arm arm) {
```
It makes it protected. U want only arm the create arms commands builders